example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -398,7 +400,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Is there a reason those ios related changes are included ?

Reply:
Yes i use mac with last version of xcode and auto change it not manually to run app in simulator

Reply:
It is in example project not in library code

Reply:
In general a PR would contain the changes required for an use case. There seems to be unrelated changes here. Personally I would split it in multiple PR to indicate what is the intent of each of those.

Reply:
I don't understand